pkg/cmd/build/build.go

```diff
@@ -156,6 +156,10 @@ func (ob *OktetoBuilder) buildWithOkteto(ctx context.Context, buildOptions *type
 
 	reg := registry.NewOktetoRegistry(GetRegistryConfigFromOktetoConfig(ob.OktetoContext))
 
+	if err := buildkitWaiter.WaitUntilIsUp(ctx); err != nil {
```

Comment:
If we are calling it, and waiting here, would it make sense to remove the call we do within `buildSolver.Run`? or should we keep both in case we call to `Run` from another place?

Reply:
I'd keep it in case there is a restart in the middle of both events but we can remove it also.